pkg/cri/cri.go

```diff
@@ -97,7 +99,13 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 		return nil, fmt.Errorf("failed to create containerd client: %w", err)
 	}
 
-	s, err := server.NewCRIService(c, client)
+	var s server.CRIService
+	if os.Getenv("ENABLE_CRI_SANDBOXES") != "" {
```

Comment:
possibly a set of sandboxed options here? or just "sandboxed" (sbserver)?

Reply:
I was thinking of this more like a boolean switch (originally I had `strconv.ParseBool`). But then simplified it to just empty/non-empty check. What kind of options can we expect here?